src/scripts/snabb_bot.sh

```diff
@@ -26,13 +26,18 @@ function init {
     fi
     export logdir="$SNABBBOTDIR/log"
     export tmpdir="$SNABBBOTDIR/tmp"
+    rm -Rf "$SNABBBOTDIR" "$logdir" "$tmpdir"
```

Comment:
`$logdir` is actually supposed to persist (it caches results, otherwise everything is recomputed all the time).